robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -405,4 +469,103 @@ public static final class CONTROLLER {
     public static final double DRIVE_RUMBLE_SECONDS = 2;
     public static final double CODRIVE_RUMBLE_SECONDS = 2;
   }
+
+  public static class FIELD { // pulled directly from Choreo or field drawings provided by FIRST
+
+    public static class REEF {
```

Comment:
Move reef side constants into here, and rename `BRANCH` constants and rename constants to be related to eachother.